enterprise/tasks.py

```diff
@@ -182,3 +208,130 @@ def send_sso_configured_email(
         )
         LOGGER.exception(message)
         raise exc
+
+
+@shared_task
+@set_code_owner_attribute
+def send_group_membership_invitation_notification(
+    enterprise_customer_uuid,
```

Comment:
instead of the uuid, can we just accept the customer record? Since we'll have that in the view

Reply:
Celery tasks generally don't deal well with accepting model instances as arguments - the general pattern is to pass in object identifiers, and then fetch those within the task.  This is because task name, options, and input has to be serialized (with either json or pickle) before its sent to the broker and then to the worker.